_posts/2020-06-03-virtual-workshop-recap.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: Virtual US-RSE Workshop Recap
+tags: [workshop]
+---
+
+by Jordan Perr-Sauer, Vanessa Sochat, Ian Cosden, Charles Ferenbaugh, Daniel S. Katz, Sandra Gesing
```

Comment:
For authors, did you consider (and preview `posted_by`) with a list, so it renders in the author space for the post? Another idea if that doesn't look good is to have this be in quotes:

> by Jordan Perr ... Gesing

I think we can improve upon having it just as a line of text. Here is an example of postedby:

```
---
layout: post
title: US-RSE Goals 
subtitle: US-RSE Steering Committee sets initial goals
posted_by: Ian Cosden
tags: [governance]
---
```

Reply:
Done. List didn't work. Used a string separated by commas. I had copied the template for this post from the 2020-04-09-virtual-workshop.md post, which has no posted_by.